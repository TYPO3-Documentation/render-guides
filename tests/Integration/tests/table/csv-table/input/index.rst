=========
CSV table
=========

..  csv-table:: Numbers
    :header: "Header 1", "Header 2"
    :widths: 15, 15

    1, "one"
    2, "two"
