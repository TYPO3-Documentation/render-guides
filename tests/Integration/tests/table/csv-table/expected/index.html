<!-- content start -->
                <section class="section" id="csv-table">
            <h1>CSV table<a class="headerlink" href="#csv-table" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline"><i class="fa-solid fa-paragraph"></i></a></h1>
            <div class="table-responsive">
    <table class="table table-bordered align-default colwidths-grid">
                    <colgroup>
                                    <col style="width: 15%">
                                                <col style="width: 15%">
                        </colgroup>
            <thead>
            <tr>
                            <th>Header 1</th>
                            <th>Header 2</th>
                    </tr>
        </thead>
        <tbody>
    <tr>
            <td>1</td>            <td>one</td>    </tr>    <tr>
            <td>2</td>            <td>two</td>    </tr></tbody>    </table>
</div>
    </section>
        <!-- content end -->
